testutils/infra/container_manager/docker_compose_manager.py

```diff
@@ -109,7 +109,7 @@ class DockerComposeNamespace(DockerNamespace):
     ]
 
     NUM_SERVICES_OPENSOURCE = 13
-    NUM_SERVICES_ENTERPRISE = 16
+    NUM_SERVICES_ENTERPRISE = 17
```

Comment:
We need a similar update in the mender-docs tests See `EXPECTED_COUNT_ENTERPRISE` in [here](https://raw.githubusercontent.com/mendersoftware/mender-docs/master/07.Server-installation/03.Installation-with-docker-compose/02.Upgrading-from-OS-to-Enterprise/docs.md).